scripts/systemd/xcash-dpops.service

```diff
@@ -1,13 +1,15 @@
 [Unit]
 Description=X-Cash DPOPS Program background process
+After=network.target xcash-rpc-wallet.service MongoDB.service
```

Comment:
Can you apply these changes to the embedded systemd service files inside the autoinstaller? Since these are just templates for show and not the actual templates that get installed on the system, its the ones inside the autoinstaller